rs/nervous_system/chunks/src/lib.rs

```diff
@@ -1,8 +1,43 @@
 use ic_crypto_sha2::Sha256;
 use ic_stable_structures::{Memory, StableBTreeMap};
+use lazy_static::lazy_static;
 
 const CHUNK_SIZE_BYTES: usize = 1_800_000;
 
+lazy_static! {
```

Comment:
If it's intended only for tests, it should be in a module called that has "test" in the name.   It is good to communicate intent.

Reply:
Comments aren't as good as variable and module names.  It's very easy to use code without reading all the comments.

Reply:
I don't see how these would be misused if someone mistakenly thought, "these would be really useful in production".